weblate/trans/models/component.py

```diff
@@ -931,7 +931,7 @@ def install_autoaddon(self):
                 continue
 
             if (
-                addon.project_scope
+                getattr(addon, "project_scope", False)
```

Comment:
This really should not be needed. `addon` here should be a `BaseAddon` subclasss which has this attribute.